DedicatedServer/Scripts/Libs/Zrx/ModeLibs/FlagRush/UI/Modules/Menu.Script.txt

```diff
@@ -0,0 +1,105 @@
+// #RequireContext CSmMlScriptIngame
+
+Text GetManialink() {
+	return """
+	<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
+	<manialink version="3">
+	<stylesheet>
+	<style class="text" textsize="2" textfont="GameFontRegular" valign="center2" />
+	<style class="header" textsize="2" textfont="GameFontSemiBold" valign="center2" />
+	</stylesheet>
+	<frame z-index="5000" id="main-menu" hidden="0">
+	<quad  id="background" z-index="-2" pos="0 0" size="70 80" opacity="0.9" alphamask="file://Media/Manialinks/Nadeo/TMNext/Menus/PageMatchmakingMain/Matchmaking_lobby_mode_mask.dds" image="file://Media/Manialinks/Nadeo/TMNext/Menus/Background_marketing.dds" halign="center"  modulatecolor="115EFFF" valign="center"/>
+	<quad pos="0 28" z-index="0" size="50 0.33" bgcolor="FFF" halign="center" valign="top"/>
+	<label pos="0 34" z-index="0" size="93.6 8" text="FlagRush" textfont="GameFontRegular" textsize="6" halign="center" valign="center"/>
+	<frame pos="0 15" class="uiContainer uiButton">
+	<quad size="50 8" style="Bgs1InRace" class="" substyle="BgColorContour" halign="center" valign="center2" />
+	<label size="49 7" text="Resume" class="uiButtonElement" halign="center" valign="center2" focusareacolor1="0000" focusareacolor2="3ada" scriptevents="1" translate="0" textsize="1.2" data-action="Resume"/>
+	</frame>
+	<frame pos="0 5" class="uiContainer uiButton">
+	<quad size="50 8" style="Bgs1InRace" class="" substyle="BgColorContour" halign="center" valign="center2" />
+	<label size="49 7" text="Change Team" class="uiButtonElement" halign="center" valign="center2" focusareacolor1="0000" focusareacolor2="3ada" scriptevents="1" translate="0" textsize="1.2" data-action="ChangeTeam"/>
+	</frame>
+	<frame pos="0 -5" class="uiContainer uiButton">
+	<quad size="50 8" style="Bgs1InRace" class="" substyle="BgColorContour" halign="center" valign="center2" />
+	<label size="49 7" text="Spectate" class="uiButtonElement" halign="center" valign="center2" focusareacolor1="0000" focusareacolor2="3ada" scriptevents="1" translate="0" textsize="1.2" data-action="Spectate" />
+	</frame>
+	<frame pos="0 -15" class="uiContainer uiButton">
+	<quad size="50 8" style="Bgs1InRace" class="" substyle="BgColorContour" halign="center" valign="center2" />
+	<label size="49 7" text="Setup..." class="uiButtonElement" halign="center" valign="center2" focusareacolor1="0000" focusareacolor2="3ada" scriptevents="1" translate="0" textsize="1.2" data-action="Setup" />
```

Comment:
Should be renamed like "Modesettings" or something.

Reply:
fixed.